blog/src/components/Navbar/Navitem.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Navitem(props) {
+  return (
+    <li className="nav-item">
+    <a href={props.href} className="nav-link">{props.text}{props.children}</a>
+  </li>
```

Comment:
It would be awesome to use [react-router](https://reactrouter.com/en/main) to navigate between pages.